lang/en/lychee.php

```diff
@@ -315,6 +315,7 @@
 	'U2F_AUTHENTIFICATION_SUCCESS' => 'Authentication successful!',
 	'U2F_CREDENTIALS' => 'Credentials',
 	'U2F_CREDENTIALS_DELETED' => 'Credentials deleted!',
+	'U2F_LOGIN' => 'Login with WebAuthn',
```

Comment:
With my "grammar Nazi" hat on, I believe this should actually be:

```suggestion
	'U2F_LOGIN' => 'Log in with WebAuthn',
```

(verb vs noun)

Reply:
Fixed in [cba3849](https://github.com/LycheeOrg/Lychee/pull/2155/commits/cba3849efb6514c55b72d865a60d1961525506de)